app/templates/partials/widgets/percentage_widget.html

```diff
@@ -26,6 +26,6 @@
 
     {% include 'partials/forms/input.html' %}
 
-    <span class="input-type__type" id="{{answer.id}}-type">%</span>
+    <abbr title="%" class="input-type__type" id="{{answer.id}}-type">%</abbr>
```

Comment:
You'll need to adjust the test in `tests/integration/components/test_render_percentage_widget.py` to use abbr instead of span to get travis working. There may be other instances where span has been used in the tests.

Reply:
Will do, I'll dig through to find them